src/core.js

```diff
@@ -748,7 +748,7 @@ define([
         if (target) {
             // the .change fires the validation controls
             if (!widgets.util.getWidget(target, this).options.noRichText &&
-                ((!mug && _this.data.core.form.useRichText !== false &&
+                ((!mug && _this.data.core.form.noRichText !== false &&
```

Comment:
I'm not sure what I was thinking when I wrote this if statement, it took me a minute to get my head around.

What do you think about adding `noRichText` to the base widget? And it could be overwritten by `richText` widget. That way we never need to look at the form, mug or opts anymore